migrations/tables/instance-report.yaml

```diff
@@ -35,6 +35,8 @@ spec:
         type: text
       - name: kurl_install_id
         type: text
+      - name: embedded_cluster_id
+        type: text
```

Comment:
do we need to add to this table anymore?

Reply:
not sure, but wouldn't hurt. we can remove this table altogether later.